back/engines/commercial/analytics/spec/factories/dimension_locales.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :dimension_locale_en, class: 'Analytics::DimensionLocale' do
+    name { 'en' }
+  end
+  factory :dimension_locale_nl, class: 'Analytics::DimensionLocale' do
+    name { 'nl' }
+  end
```

Comment:
I suggest only one factory here:
```suggestion
  factory :dimension_locale, class: 'Analytics::DimensionLocale' do
    name { 'en' }
  end
```

In specs, you can create the `nl` version with:
```ruby
create(:dimension_locale, name: 'nl')
```